doc/getting_started/index.rst

```diff
@@ -179,30 +159,23 @@ We'll also install Zephyr's multi-purpose west tool.
 
             choco feature enable -n allowGlobalConfirmation
 
-      #. Install CMake:
+      #. Use ``choco`` to install dependencies:
 
          .. code-block:: console
 
             choco install cmake --installargs 'ADD_CMAKE_TO_PATH=System'
+            choco install ninja gperf python git
 
-      #. Install the rest of the tools:
-
-         .. code-block:: console
-
-            choco install git python ninja dtc-msys2 gperf
```

Comment:
@mbolivar-nordic I'm sure this is documented somewhere (as otherwise we'd have new users complaining all day long), but I wonder how windows users are supposed to get dtc if not documented here.

Reply:
> @mbolivar-nordic I'm sure this is documented somewhere (as otherwise we'd have new users complaining all day long), but I wonder how windows users are supposed to get dtc if not documented here.

It's not required; 23df708aa36d9b748b54d1c362e7e2e67316593c

Reply:
> I'm sure this is documented somewhere 

https://docs.zephyrproject.org/latest/guides/dts/intro.html#scripts-and-tools

Reply:
Ho, right! 
I was confused about dtc status, I knew it was still in use (at least on my side).
That's clear now. Txs @mbolivar-nordic